docs/theming-guide.adoc

```diff
@@ -4184,18 +4184,24 @@ The keys in this category control the arrangement and style of tables and table
 (default: $table-border-color)
 |table:
   grid-color: #eeeeee
+table:
+  grid-color: [eeeeee,aaaaaa]
```

Comment:
Don't add duplicate examples. That will cause confusion. The additional type should be addressed in the second column. So it would read ass:

```
<<colors,Color>> {vbar} <<colors,Color[x,y]>>
```

Reply:
That works, but I like `<<colors,Color>> {vbar} [<<colors,x>>,<<colors,y>>]` better.  What do you think?

Reply:
It's a nice syntax, but not consistent with what we've done elsewhere in the theming guide. If we want to make that change, it would need to be in a separate issue so we can apply it universally.

Reply:
So would you like me to go back to <<colors,Color>> {vbar} <<colors,Color[x,y]>> so we can close this issue?

Reply:
Yes please.